src/pocketmine/level/format/mcregion/McRegion.php

```diff
@@ -85,9 +85,9 @@ public static function generate($path, $name, $seed, $generator, array $options
 			"initialized" => new ByteTag("initialized", 1),
 			"GameType" => new IntTag("GameType", 0),
 			"generatorVersion" => new IntTag("generatorVersion", 1), //2 in MCPE
-			"SpawnX" => new IntTag("SpawnX", 128),
+			"SpawnX" => new IntTag("SpawnX", 255),
```

Comment:
Then why was it 128 before?

There are 512 blocks horizontally, so you want the coordinate between the 256th and 257th, i.e. index 255 and 256. The middle of the index-255 block is 255.5, so this should be 256.